src/client/Microsoft.Identity.Client/Internal/Requests/IntegratedWindowsAuthRequest.cs

```diff
@@ -38,9 +38,22 @@ protected override async Task<AuthenticationResult> ExecuteAsync(CancellationTok
             await ResolveAuthorityAsync().ConfigureAwait(false);
             await UpdateUsernameAsync().ConfigureAwait(false);
             var userAssertion = await FetchAssertionFromWsTrustAsync().ConfigureAwait(false);
-            var msalTokenResponse = await SendTokenRequestAsync(
-                                            GetAdditionalBodyParameters(userAssertion), cancellationToken)
-                                        .ConfigureAwait(false);
+
+            MsalTokenResponse msalTokenResponse;
+            try
+            {
+                msalTokenResponse = await SendTokenRequestAsync(
```

Comment:
I'm pretty sure `SendTokenRequestAsync` does the right thing, this is not needed. The bug states "improve error messages when Mex document cannot be parsed and when UserRealm endpoint cannot be reached", not when contacting the /token endpoint.

Reply:
removing try catch